src/components/appNavigation/TabsView.js

```diff
@@ -116,7 +116,7 @@ const TabsView = React.memo((props: TabViewProps) => {
 
   return (
     <Appbar.Header dark>
-      <TouchableOpacity onPress={goToRewards} style={{ marginLeft: '10px' }}>
+      <TouchableOpacity onPress={goToRewards} style={{ marginLeft: 10 }}>
```

Comment:
move styles  into `withStyles(getStylesFromProps)`

Reply:
@AlexeyKosinski it is already done in tother PR. So it will be ,erged to the master.